buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -441,43 +446,47 @@ def get_summary_table(identifier) {
         buildReleases.each { release ->
             // table cell
             def innerTable = "<table><tbody>"
+            def downstreamJobNames = buildFile.get_downstream_job_names(spec, release, identifier)
+            def downstreamBuilds = [:]
+            def pipelineName = ''
+            def build = null
+            def pipelineLink = '&nbsp;'
+            def pipelineRuntime = '&nbsp;'
```

Comment:
Can we change this variable name to `pipelineRunTime` instead of `pipelineRuntime`? Or even something else like duration?

Reply:
`pipelineDuration` it is